src/Writer/CsvWriter.php

```diff
@@ -65,6 +65,9 @@ final class CsvWriter implements TypedWriterInterface
      */
     private $terminate;
 
+    /**
+     * @throws \RuntimeException
```

Comment:
Maybe it's better to add this (or a more specific domain exception) to the interface?

Reply:
You mean adding

```
    /**
     * @throws \RuntimeException
     */
    public function __construct();
```
?

But how can I do this since the number of arguments is not the same ?

Reply:
A `RuntimeException` should be okay for the constructor, but the other methods (`open`, `write`, ..) should throw a `WriterException`

Reply:
The PR is updated